iocore/net/SSLCertLookup.cc

```diff
@@ -130,8 +132,7 @@ struct SSLContextStorage {
 
   /// We can only match one layer with the wildcards
   /// This table stores the wildcarded subdomain
-  InkHashTable *wilddomains;
-
+  mutable HashMap<cchar *, StringHashFns, const int *> wilddomains;
```

Comment:
the int value here is an index to the vector of contexts. The hashmap find method returns 0 in case a key is not found, which is a valid index. So I kept it pointer so that the returned 0 (i.e. NULL) means that the key is not present in the map. I considered putting a find_key method in the map but calling that every time looked expensive